addons/hr_holidays/models/hr_leave_accrual_plan_level.py

```diff
@@ -168,13 +162,13 @@ def _compute_level(self):
 
     @api.depends('first_day', 'second_day', 'first_month_day', 'second_month_day', 'yearly_day')
     def _compute_days_display(self):
-        days_select = _get_selection_days(self)
+        days_select = DAY_SELECT_SELECTION
         for level in self:
-            level.first_day_display = days_select[min(level.first_day - 1, 28)][0]
-            level.second_day_display = days_select[min(level.second_day - 1, 28)][0]
-            level.first_month_day_display = days_select[min(level.first_month_day - 1, 28)][0]
-            level.second_month_day_display = days_select[min(level.second_month_day - 1, 28)][0]
-            level.yearly_day_display = days_select[min(level.yearly_day - 1, 28)][0]
+            level.first_day_display = days_select[min(level.first_day - 1, 28)][1]
+            level.second_day_display = days_select[min(level.second_day - 1, 28)][1]
+            level.first_month_day_display = days_select[min(level.first_month_day - 1, 28)][1]
+            level.second_month_day_display = days_select[min(level.second_month_day - 1, 28)][1]
+            level.yearly_day_display = days_select[min(level.yearly_day - 1, 28)][1]
```

Comment:
This is not correct.
You should always assign the key part (first element of the tuple) of one of the possible values to a Selection field.

Reply:
:ok_hand: after further reading now i understand why we translate, it's for showing in selection